git-branchless/src/commands/reword.rs

```diff
@@ -522,6 +527,14 @@ fn parse_bulk_edit_message(
         commits_oids.insert(commit.get_short_oid()?, commit.get_oid());
     }
 
+    let message = if commits.len() == 1 && !message.contains("++ reword") {
+        // For single commits, add the marker line, but only if the user hasn't already done so.
```

Comment:
If the user has added their own marker line (however unlikely that may be), adding another one will generate a less-direct error message. So this check for `contains` is pretty edge-casey, but will end up producing a nicer error message if it ever does happen.